mjpc/tasks/humanoid/skateboard/steering.cc

```diff
@@ -164,10 +164,10 @@ void move_mocap_poses(mjtNum *result, const mjModel *model, const mjData *data,
   mju_scl(modified_mocap_pos.data(), model->key_mpos + 3 * model->nmocap * mode,
           1, 3 * (model->nmocap - 1));
   double skateboard_center[3];
-  int skateboard_body_id_ = mj_name2id(model, mjOBJ_XBODY, "skateboard");
+  int skateboard_body_id = mj_name2id(model, mjOBJ_XBODY, "skateboard");
```

Comment:
The `_` suffix is reserved for the class attributes so let's name local variables without `_`.